teachertool/src/components/RubricWorkspace.tsx

```diff
@@ -109,7 +124,7 @@ interface IProps {}
 export const RubricWorkspace: React.FC<IProps> = () => {
     return (
         <div className={css.panel}>
-            <Toolbar left={<WorkspaceTabButtons />} right={<WorkspaceToolbarButtons />} />
+            <Toolbar left={<WorkspaceTabButtons />} center={<WorkspaceAutorun />} right={<WorkspaceToolbarButtons />} />
```

Comment:
When right-justified, its position will shift left and right when switching tabs due to the changing number of buttons to the right of it, felt jarring. Always happy to run by designers.

Reply:
Hmm, that's fair, but left-justified it looked like it should be an actual selected tab to me. Could we position it on the right such that any optional buttons are to the left of it, so it won't shift? Or we can leave as-is and get feedback.

Reply:
Yeah, I think having the toggle be between the action menu and the play button is a good place for it, it also has association that way, too. The "play" button is for running the validation, and you can decide whether or not you want the validation to run automatically right next to it.

Another thought, what would people think of putting the toggle in the actions menu? I know it wouldn't be the greatest to keep something like that hidden, but I feel no matter where you put the toggle in the tab bar, it will make the bar look a bit crowded. 

Fine to wait for design feedback on this, just wanted to share my thoughts here.

Reply:
I hadn't considered putting it before the variable button set. That's a nice solution. Let me put up a new PR with that change....